Makefile

```diff
@@ -36,14 +36,19 @@ dev: build check test
 ci: build check basic-test
 
 build: export GO111MODULE=on
-build:
+build: pd-server pd-ctl pd-tso-bench pd-recover
+pd-server:
 ifeq ("$(WITH_RACE)", "1")
 	CGO_ENABLED=1 go build -race -ldflags '$(LDFLAGS)' -o bin/pd-server cmd/pd-server/main.go
 else
 	CGO_ENABLED=0 go build -ldflags '$(LDFLAGS)' -o bin/pd-server cmd/pd-server/main.go
 endif
+
+pd-ctl:
```

Comment:
I think you want to move `export GO111MODULE=on` to binary targets. Or `make pd-ctl` will not work as previous.

Reply:
I didn't find any problem because I build pd out of the GOPATH in which case the go mod is used by default.

Thanks for pointing out.